portfolio/src/main/webapp/images.html

```diff
@@ -229,6 +229,9 @@ <h2>June 2018-Present</h2>
           </a>
         </div>
       </div>
+      <div class="buttons-row">
+        <button class="button"><a href="index.html" aria-label="Back">Back</a></button>
```

Comment:
Can just remove the `aria-label` because the text content is already the same, so it will be used by default.